docs/docs/contributing-guide/setup/ubuntu.md

```diff
@@ -6,13 +6,21 @@ title: Ubuntu
 # Ubuntu
 Follow these steps to setup and run ToolJet on Ubuntu. Open terminal and run the commands below.
 
+:::info
+Clone the GitHub repo locally using: 
+
+```bash
+git clone https://github.com/ToolJet/ToolJet.git
+```
+:::
+
 ## Setting up
 
 1. Set up the environment
 
     1.1 Install Node.js
     ```bash
-    curl -sL https://deb.nodesource.com/setup_18.x | sudo -E bash -
+    curl -sL https://deb.nodesource.com/setup_18.3.0 | sudo -E bash -
```

Comment:
we can use nvm and install node since this link does work for downloading a specific version - 

   # Install NVM
    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
    # Close and reopen your terminal to start using nvm
    nvm install 18.3.0